website/docs/r/automation_job_schedule.html.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_automation_job_schedule"
+sidebar_current: "docs-azurerm-resource-automation-job-schedule"
+description: |-
+  Links an Automation Runbook and Schedule.
+---
+
+# azurerm_automation_job_schedule
+
+Links an Automation Runbook and Schedule.
+
+## Example Usage
+
+```hcl
```

Comment:
Could we just show the basic usage with only the resource here?

A complete example can go in the examples folders if you'd like, with a link from here:
```
A full example can be found [here](url)

Reply:
Done :)